### ISSUE TITLE
Can't add mrtjpcore

### ISSUE THREAD

**Issue by ZotyDev** [juraj-hrivnak/Pakku#13]
Running `pakku add mrtjpcore` results in the following error:

```
Exception in thread "main" kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 813: Expected start of the object '{', but had 'n' instead at path: $.data[0].logo
JSON input: .....e.com/members/mrtjp"}],"logo":null,"screenshots":[],"mainFil.....
        at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24)
        at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:32)
        at kotlinx.serialization.json.internal.AbstractJsonLexer.fail(AbstractJsonLexer.kt:598)
        at kotlinx.serialization.json.internal.AbstractJsonLexer.fail$default(AbstractJsonLexer.kt:596)
        at kotlinx.serialization.json.internal.AbstractJsonLexer.fail$kotlinx_serialization_json(AbstractJsonLexer.kt:233)
        at kotlinx.serialization.json.internal.AbstractJsonLexer.fail$kotlinx_serialization_json$default(AbstractJsonLexer.kt:228)
        at kotlinx.serialization.json.internal.AbstractJsonLexer.unexpectedToken(AbstractJsonLexer.kt:225)
        at kotlinx.serialization.json.internal.StringJsonLexer.consumeNextToken(StringJsonLexer.kt:74)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.beginStructure(StreamingJsonDecoder.kt:102)
        at teksturepako.pakku.api.models.cf.CfModModel$ModAsset$$serializer.deserialize(CfModModel.kt:94)
        at teksturepako.pakku.api.models.cf.CfModModel$ModAsset$$serializer.deserialize(CfModModel.kt:94)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
        at teksturepako.pakku.api.models.cf.CfModModel$$serializer.deserialize(CfModModel.kt:34)
        at teksturepako.pakku.api.models.cf.CfModModel$$serializer.deserialize(CfModModel.kt:34)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
        at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:538)
        at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
        at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
        at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
        at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
        at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
        at teksturepako.pakku.api.models.cf.SearchProjectResponse$$serializer.deserialize(CfCommunications.kt:12)
        at teksturepako.pakku.api.models.cf.SearchProjectResponse$$serializer.deserialize(CfCommunications.kt:12)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        at kotlinx.serialization.json.Json.decodeFromString(Json.kt:107)
        at teksturepako.pakku.api.platforms.CurseForge.requestProjectFromSlug(CurseForge.kt:319)
        at teksturepako.pakku.api.platforms.CurseForge$requestProjectFromSlug$1.invokeSuspend(CurseForge.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at teksturepako.pakku.cli.cmd.Add.run(Add.kt:22)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:306)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:319)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:40)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:458)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:455)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:475)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:482)
        at teksturepako.pakku.MainKt.main(Main.kt:24)
```

**Comment by juraj-hrivnak** [comment]
It looks like the project logo on CurseForge can be null.

**Comment by juraj-hrivnak** [comment]
Fixed in v0.11.1
